elivepatch_server/resources/livepatch.py

```diff
@@ -43,7 +43,7 @@ def build_livepatch(self, vmlinux, jobs, debug=True):
                        'main.patch']
         if debug:
             bashCommand.extend(['--skip-cleanup'])
-            bashCommand.extend(['--debug'])
+            bashCommand.extend(['--dddd'])
```

Comment:
Shouldn't this be single dash, i.e. `-dddd` per https://github.com/dynup/kpatch/pull/760/files#diff-c4bc518fff9b8f8607f5f407c34c8809R381